tests/test_fields.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+
+from bankline_parser.data_services import fields
+from bankline_parser.data_services import enums
+from bankline_parser.data_services.exceptions import ParseError
+
+
+class DataServicesFieldsTestCase(unittest.TestCase):
```

Comment:
This is a good set of tests :+1:

I might prefer them to be grouped into per-field cases (`DataServicesTextFieldTestCase`) with test names that are a bit more sentence-like (`test_parsing_basic_text_field_succeeds`, `test_parsing_left_justified_text_field_with_padding_strips_padding`), but that's a minor detail (and one I know we've disagreed on every time I mention it :wink:).